modules/carto/src/layers/vector-tile-layer.ts

```diff
@@ -47,6 +46,12 @@ export default class VectorTileLayer<ExtraProps extends {} = {}> extends MVTLaye
     mvt: boolean;
   };
 
+  constructor(props: VectorTileLayerProps) {
```

Comment:
all other layers support many props and they are somehow merged
i guess, to be compatible with other layers, it should be something like

```
constructor(...props: VectorTileLayerProps[]) { 
  super(...props)
}
```

Reply:
Good point